scripts/repl.js

```diff
@@ -1,5 +1,15 @@
 (function(babel, $, _, ace, window) {
 
+  /* Throw meaningful errors for getters of commonjs. */
+  ["module", "exports", "require"].forEach(function(commonVar){
+    Object.defineProperty(window, commonVar, { 
+      get: function () { 
+      throw new Error(commonVar + " is not supported in the browser,\
+        you need a commonjs environment such as node.js/io.js, browserify/webpack etc");
```

Comment:
Looks like this for me.

![screen shot 2015-02-19 at 10 22 30 pm](https://cloud.githubusercontent.com/assets/952783/6282211/dab9a09c-b885-11e4-98cf-bcd65bec1ddd.png)

I've considered writing it in ES6, but its a small codebase and I don't feel like adding a build step (makes it super easy to just merge) :stuck_out_tongue_winking_eye: